demo/browser.json

```diff
@@ -17,6 +16,12 @@
         "../src/components/ebay-notice",
         "../src/components/ebay-pagination",
         "../src/components/ebay-select",
-        "../src/components/ebay-radio"
+        "../src/components/ebay-radio",
+        {
+            "if-flag": "ds6",
```

Comment:
Wouldn't the flag be `skin-ds6`? Did you verify this locally?

Reply:
Good catch Yoni. I think you're right but will let Sean verify.